benchmark/doublecoset/doublecoset2.tst

```diff
@@ -0,0 +1,18 @@
+#############################################################################
+##
+#W  doublecoset.tst                                     Alexander Hulpke
+##
+##
+#Y  Copyright (C)  1997,  Lehrstuhl D für Mathematik,  RWTH Aachen,  Germany
+##
+##  This  file  tests for double coset calculations
+##
+gap> START_TEST("doublecoset.tst");
```

Comment:
Shouldn't this be `doublecoset2` (though I don't see the merit in splitting the tests up so much